5_analyze_data/notebooks/UMAP_analysis/scripts/add_singlecell_count_to_csv.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
```

Comment:
We had previously discussed removing this file (and resetting the original `nf1_sc` csvs

Reply:
It's not good practice to rewrite previously created files!

Reply:
Definitely! Probably would have been better to do this in the feature extraction module as adding we are adding metadata that wasn't added from CP.

This file has been moved to be ignored.